### DISCUSSION TITLE
Name controller in plural form

### DISCUSSION THREAD

**Discussion by SergeyXXI** [yiisoft/yii2#20264] [Q&A] [answered]
Hello, didn't find anywhere about that. Is it normal to use plural form naming for controllers? For example, "CategoriesController" instead of "CategoryController"? Or is it bad practice?
If I wanna see "categories" in address bar, but using CategoryController, then some extra work need to be done(which I don't want ideally): add urlManager rule like"categories" => "category/index" and maybe some rewrite rule for web server redirecting "category" to "categories".

**Comment by samdark** [comment, marked as answer]
Whatever works. No special restrictions about it.
